### PR TITLE
Create HUD Product Lead and Technical Architect files

### PR DESCRIPTION
**Pull request by pixelwake**

### CODE REVIEW

File: _positions/hud-fellow-product-lead.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+#User Experience Designer, Information Architecture
+
+##Basic Information:
+
+**Agency:** [Department of Housing and Urban Development (HUD)](http://portal.hud.gov/hudportal/HUD)
+
+**Position Type:** Excepted Service - Time-Limited Appointment - Not to exceed 24 months; may be extended for a maximum of 4 years total
+
+**Compensation/Grade Level:** GS-14.GS-15
```
Comment (fureigh) [inline _positions/hud-fellow-product-lead.md]: What's the period about? Same question as on another pull request.

File: _positions/hud-fellow-product-lead.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+#User Experience Designer, Information Architecture
+
+##Basic Information:
+
+**Agency:** [Department of Housing and Urban Development (HUD)](http://portal.hud.gov/hudportal/HUD)
+
+**Position Type:** Excepted Service - Time-Limited Appointment - Not to exceed 24 months; may be extended for a maximum of 4 years total.
+
+**Compensation/Grade Level:** GS-14.GS-15
```
Comment (fureigh) [inline _positions/hud-fellow-product-lead.md]: Is this period intentional?

File: _positions/hud-fellow-product-lead.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- [Information regarding the General Schedule Grade Levels](https://www.opm.gov/policy-data-oversight/pay-leave/pay-systems/general-schedule/0)
+- [General Schedule Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/)
+
+**Key Requirements:** U.S. citizenship is required.
+
+**Security Clearance:** Public Trust - Background Investigation Required
```
Comment (fureigh) [inline _positions/hud-fellow-product-lead.md]: Is this intentionally all capitalized?

File: _positions/hud-fellow-product-lead.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+**Benefits:** The federal government offers a comprehensive benefits package. [Explore the major benefits] (http://www.usajobs.gov/ResourceCenter/Index/Interactive/Benefits#icc) offered to most federal employees. 
+
+Please visit [HUD Careers](http://portal.hud.gov/hudportal/HUD?src=/program_offices/administration/careers) for more information about what it’s like to work at HUD.
+ 
+##Role Summary:
+The Product Lead tech fellow at the U.S. Department of Housing and Urban Development (HUD) is responsible for managing the development, delivery, and continuous improvement of internal products. As a Product Lead, you’ll work with members of HUD to evaluate potential work, lead a cross-functional team of other tech fellows, and set the vision for all products.
+
+##Key Objectives:
+**You will lead a cross-functional team to deliver the right product using agile-based methodologies.**
+- Build meaningful relationships with partners at HUD and help them understand user centered, iterative approach to understanding the problem and building the solution
```
Comment (fureigh) [inline _positions/hud-fellow-product-lead.md]: Hyphenate `user-centered`

File: _positions/hud-fellow-technical-architect.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+*Please note: This role is for the US Department of Housing and Urban Development's innovation fellowship program, not 18F.*
+
+#Technical Architect
+
+##Basic Information:
+
+**Agency:** [Department of Housing and Urban Development (HUD)](http://portal.hud.gov/hudportal/HUD)
+
+**Position Type:** Excepted Service - Time-Limited Appointment - Not to exceed 24 months; may be extended for a maximum of 4 years total
```
Comment (fureigh) [inline _positions/hud-fellow-technical-architect.md]: Same questions as above. And is this position in D.C. or distributed?

File: _positions/hud-fellow-technical-architect.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+**Compensation/Grade Level:** GS-14.GS-15
```
Comment (fureigh) [inline _positions/hud-fellow-technical-architect.md]: Should this hyphen be a slash?

File: _positions/hud-fellow-technical-architect.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+- [Information regarding the General Schedule Grade Levels](https://www.opm.gov/policy-data-oversight/pay-leave/pay-systems/general-schedule/0)
+- [General Schedule Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/)
+
+**Key Requirements:** U.S. Citizenship is required.
```
Comment (fureigh) [inline _positions/hud-fellow-technical-architect.md]: Lowercase `citizenship`

File: _positions/hud-fellow-technical-architect.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+**Security Clearance:** Public Trust - Background Investigation Required
```
Comment (fureigh) [inline _positions/hud-fellow-technical-architect.md]: Capitalization?

**Comment by fureigh** [comment]
Back to you, @rooneywp.

**Comment by pixelwake** [comment]
@fureigh I addressed your comments. It should be good to go now.

**Comment by fureigh** [comment]
Thanks, @rooneywp! Taking another look now.

**Comment by fureigh** [comment]
404 caveat from #310 applies. Merging for now and then reapplying to `rooneywp-patch-9`.

**Pull request merged**